server/spec/test.js

```diff
@@ -0,0 +1,19 @@
+const request = require('request');
+
+test('Able to retrive one data given ID', () => {
+  request('http://127.0.0.1:3000/api/turash/description/50', function(err, response, body) {
+    var output = JSON.parse(body);
+    expect(typeof output).toBe('object');
+    expect(output.id).toBe(50);
+    expect(output).toHaveProperty('trips');
+    expect(output).toHaveProperty('faq');
+  });
+
+  request('http://127.0.0.1:3000/api/turash/description/70', function(err, response, body) {
```

Comment:
does this test actually work? I think we might be relying on jest's default timeout of 5s too much https://jestjs.io/docs/en/api#testname-fn-timeout

also, will this test fail any time you reseed the database?

Reply:
I ran the test again and it looks like it's working. I also tried reseeding the database too and all the tests ran under 5 sec.

Reply:
ah yes, I misread the test the first time and thought you were checking for the property values, but you're just checking for the existence of the property, I see!

I'd double check that the test is working OK by trying to break it. (eg, test for a property that doesn't exist, just so you know it'll throw errors if something were to go wrong in the future)

Reply:
ok!